bee-storage/bee-storage/src/access/truncate.rs

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::backend::StorageBackend;
+
+/// `Truncate<K, V>` trait extends the `StorageBackend` with `truncate` operation for the (key: K, value: V) pair;
+/// therefore, it should be explicitly implemented for the corresponding `StorageBackend`.
+#[async_trait::async_trait]
+pub trait Truncate<K, V>: StorageBackend {
+    /// Truncates all the entries associated with the (K, V) pair from the storage.
```

Comment:
What does truncation mean in this context?

Reply:
Removing all entries in a column family